app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,156 @@
+<resources>
+    <string name="app_name" translatable="false">Myne</string>
+    <!-- Diğer Metinler -->
```

Comment:
Don't translate these comments as they're basically used to easily find strings when specific sections of the UI need updates and some strings need to be modified or added. Keep them in English like the other locale files.